channels/us.m3u

```diff
@@ -3697,3 +3697,77 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
+https://pbs.lls.cdn.pbs.org/est/index.m3u8
+#EXTINF:-1 tvg-id="WJCT.us" tvg-name="WJCT-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wjct-color-cobranded-logo-HY1ohKx.png" group-title="Public Broadcasting",PBS Jacksonville (WJCT) (Geo)
+https://wjctdt.lls.cdn.pbs.org/wjctdt/5eee712c-8713-4283-8af1-c0ba9d25d6fd/primary.m3u8
+#EXTINF:-1 tvg-id="WLRN.us" tvg-name="WLRN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wlrn-color-single-brand-logo-jyMYfk8.png" group-title="Public Broadcasting",PBS Miami (WLRN) (Geo)
+https://wlrndt.lls.cdn.pbs.org/wlrndt/8f70a678-a636-4dfa-873e-699f0212f78b/primary.m3u8
+#EXTINF:-1 tvg-id="WSRE.us" tvg-name="WSRE-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wsre-color-cobranded-logo-C8DUfBI.png" group-title="Public Broadcasting",PBS Pensacola (WSRE) (Geo)
+https://wsredt.lls.cdn.pbs.org/wsredt/fc47f495-57b9-4aea-bb6b-d2ce52b74ea1/primary.m3u8
+#EXTINF:-1 tvg-id="WPBT.us" tvg-name="WPBT-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wpbt-color-cobranded-logo-QKMOlbI.png" group-title="Public Broadcasting",PBS Miami (WPBT) (Geo)
+https://wpbtdt.lls.cdn.pbs.org/wpbtdt/b2c0d4c6-4544-4bb3-8d2d-bc803c5d7381/primary.m3u8
+#EXTINF:-1 tvg-id="WXEL.us" tvg-name="WXEL-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wxel-color-cobranded-logo-q5DODDu.png" group-title="Public Broadcasting",PBS West Palm Beach (WXEL) (Geo)
+https://wxeldt.lls.cdn.pbs.org/wxeldt/6f553836-7e24-445a-8c3c-cc418aaba9dc/primary.m3u8
+#EXTINF:-1 tvg-id="WFSU.us" tvg-name="WFSU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wfsu-color-cobranded-logo-ok3uUgp.png" group-title="Public Broadcasting",PBS Tallahassee (WFSU) (Geo)
+https://wfsuhd.lls.cdn.pbs.org/wfsuhd/da7db8ae-5c3b-450e-9a25-76aa661eac79/primary.m3u8
+#EXTINF:-1 tvg-id="WEDU.us" tvg-name="WEDU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="Public Broadcasting",PBS Tampa (WEDU) (Geo)
+https://wedudt.lls.cdn.pbs.org/wedudt/c4ab80c1-ca8c-4a6e-a8b5-c5995919792b/primary.m3u8
+#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="Public Broadcasting",PBS Georgia (WVAN) (WXGA) (Geo)
+https://wgtvdt.lls.cdn.pbs.org/wgtvdt/a4907776-c769-452f-ab2f-74bb1f86676f/primary.m3u8
+#EXTINF:-1 tvg-id="WBPA.us" tvg-name="WPBA-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wpba-color-single-brand-logo-d751apM.png" group-title="Public Broadcasting",PBS Atlanta (WBPA) (Geo) (National PBS Feed)
+https://pbs.lls.cdn.pbs.org/est/index.m3u8
```

Comment:
3 different call signs but same link, is this a multicast one ?

Reply:
No there really placeholders as PBS I think is requiring streams in the near future for all affiliates.